maximum_subarray/Cpp/max_subarray.cpp

```diff
@@ -0,0 +1,28 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+int main(){
+	// Reading size of array and array.
+	cin >> n;
+	
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*